packages/ui/src/index.ts

```diff
@@ -63,6 +63,9 @@ export type { IconButtonProps } from './molecules/IconButton'
 export { default as Modal } from './molecules/Modal'
 export type { ModalProps } from './molecules/Modal'
 
+export { default as Incentives } from './molecules/Incentives'
```

Comment:
Shouldn't `IncentivesItem` be exported also?

Reply:
Definitely yes, I forgot to add that. Thanks!!